src/rgw/rgw_rest_s3.cc

```diff
@@ -6978,6 +6978,10 @@ rgw::auth::s3::STSEngine::get_session_token(const DoutPrefixProvider* dpp, const
     return -EINVAL;
   }
   string secret_s = cct->_conf->rgw_sts_key;
+  if (secret_s.empty()) {
+    ldpp_dout(dpp, 5) << "ERROR: rgw sts key not set" << dendl;
```

Comment:
i think this would be an appropriate place to use log level 0 or 1, since we'd expect an admin to configure a key

Reply:
can a bogus client spam this on a cluster not using sts?

Reply:
true. but with first-class support for roles through the account feature, don't we want clusters to support sts?

Reply:
i.e. a bogus client may just be an account user expecting AssumeRole to work